pySDC/implementations/convergence_controller_classes/collocation_inexactness.py

```diff
@@ -0,0 +1,161 @@
+import numpy as np
+
+from pySDC.core.Lagrange import LagrangeApproximation
+from pySDC.core.ConvergenceController import ConvergenceController, Status
+from pySDC.core.Collocation import CollBase
+
+
+class CollocationInexactness(ConvergenceController):
```

Comment:
Why is that called `Inexactness`?

Reply:
I think the most intuitive use for this convergence controller by itself is to gradually refine the collocation problem. I am not clear on the exact definition of inexactness, but this is what I picture: To solve an adjacent, less accurate problem during early iterations and refine to the full problem later on.
Of course you can also do the opposite here or do iterations with less resolution but the same computational cost.
Do you have suggestions for a better name?

Reply:
I guess it's rather refinement and coarsening, but since it it both people call this adaptivity. But I'm not sure if that's not even more confusing.

Reply:
How about `AdaptiveCollocation`?